qemu/tests/cfg/guest_iommu_group.cfg

```diff
@@ -0,0 +1,41 @@
+# Please add "iommu=pt intel_iommu=on" to host kernel line
+- guest_iommu_group: 
+    virt_test_type = qemu
+    type = guest_iommu_group
+    only virtio_blk virtio_net virtio_scsi
+    only q35
+    no WinXP WinVista Win7 Win8 Win8.1 Win2003
+    no Win2008 Win2008..r2 Win2012 Win2012..r2
+    no Host_RHEL.m7
+    machine_type_extra_params = "kernel-irqchip=split"
+    virtio_dev_iommu_platform = on
```

Comment:
This parameter will add `iommu_platform=on` in all virtio devices, but not all devices support this option. So you need to custom virtio_dev_filter, like `virtio_dev_filter = '^(?:(?:virtio-)|(?:vhost-))(?!(?:balloon)|(?:user)|(?:iommu))'`.